MapboxCoreNavigation/EventDetails.swift

```diff
@@ -352,7 +352,9 @@ enum EventDetailsError: Error {
 
 extension EventDetails {
     func asDictionary() throws -> [String: Any] {
-        let data = try JSONEncoder().encode(self)
+        let encoder = JSONEncoder()
+        encoder.keyEncodingStrategy = .convertToSnakeCase
+        let data = try encoder.encode(self)
```

Comment:
Remember to remove this bit, since we don’t actually want the keys to be snake case after all.

Reply:
#2628